.gitignore

```diff
@@ -0,0 +1,24 @@
+### JetBrains IDE ###
```

Comment:
:+1: on this! Would you mind adding it to github.com/pion/.goassets 

We use that to sync common files across all repos

Reply:
Will do that on the weekend when I have some time, along with all other comments. Thanks for the review, these are really useful comments :)